src/blocks/events/styles/editor.scss

```diff
@@ -54,3 +53,90 @@
 [data-type="coblocks/event-item"]:not(.is-selected) > .block-editor-block-list__block-edit::before {
 	border: 1px dashed rgba(123, 134, 162, 0.3);
 }
+
+.coblocks-events-nav-button {
+	&__prev,
+	&__next {
+		background-color: transparent;
+		background-image: url(/assets/lightbox/arrow-right.svg);
+		border: none;
+		border-radius: 100%;
+		cursor: pointer;
+		height: 62px;
+		margin: 0 1%;
+		padding: 15px;
+		position: absolute;
+		top: 50%;
+		width: 62px;
+	}
+
+	&__prev {
+		left: -4em;
+		transform: rotate(180deg);
+	}
+
+	&__next {
+		right: -4em;
+	}
+}
+
+.coblocks-events-swiper-container-external-calendar-edit div .wp-block-coblocks-events-front-container .wp-block-coblocks-front-events-swiper-container .wp-block-coblocks-events {
+
+	.wp-block-coblocks-event-item:not(:first-of-type) {
+		display: none !important;
+	}
+
+	.wp-block-coblocks-event-item {
+		pointer-events: none;
+	}
+}
+
+.coblocks-events-swiper-container-external-calendar-edit .coblocks-events-nav-button {
+	&__prev,
+	&__next {
+		top: 10%;
+	}
+}
+
+.coblocks-events-block-inner-blocks-container-edit .coblocks-events-nav-button {
```

Comment:
@snovosel no need to repeat top level class with scss. `.coblocks-events-block-inner-blocks-container-edit` is repeated below. We can simplify.